pkg/controller/autoimport/autoimport_controller.go

```diff
@@ -132,6 +136,7 @@ func (r *ReconcileAutoImport) Reconcile(ctx context.Context, request reconcile.R
 				return reconcile.Result{}, err
 			}
 			// auto import secret invalid, stop retrying
+			reqLogger.Info("Auto import secret invalid", "managedCluster", managedCluster.Name)
```

Comment:
warning is better, and this might not be quite helpful since we do not know why secret is invalid.

Reply:
Done.